apps/web/src/utils/purchase-validation.ts

```diff
@@ -8,12 +8,17 @@ import {
   minDate,
   number,
   object,
+  oneOf,
   required,
   string,
 } from 'validator-fns'
 
 import { formatCurrency } from '@/utils/format-currency'
 
+// Maximum bid for card payments as integer
+// eslint-disable-next-line unicorn/numeric-separators-style
```

Comment:
Throws the below error with `3_000_00` and this is an integer, so the disable line is needed. 
```
Invalid group length in numeric value.eslintunicorn/numeric-separators-style
```